src/Shared.TextureContainer/WebP/Texture2DExt.cs

```diff
@@ -112,13 +111,8 @@ public static unsafe byte[] LoadRGBAFromWebP(byte[] lData, ref int lWidth, ref i
 				{
 					scalingFunction(ref lWidth, ref lHeight);
 				}
-				
-				// If mipmaps are requested we need to create 1/3 more memory for the mipmaps to be generated in.
-				int numBytesRequired = lWidth * lHeight * 4;
-				if (lMipmaps)
-				{
-					numBytesRequired = Mathf.CeilToInt((numBytesRequired * 4.0f) / 3.0f);
-				}
+
+				int numBytesRequired = ImageHelper.CalculateTextureBytes(lWidth, lHeight, lMipmaps);
```

Comment:
Why not have CalculateTextureBytes right next to where it's used since it's used in only one place?

Reply:
Idea was that if anything else would need it in the future, it would already be in a convient place. Instead of in the WebP bindings. I can still move it there if you want

Reply:
If it's not required for other formats right now I'd put it here and leave it even if future formats also need such a calculation.